packages/e2e-test-utils/src/get-edited-post-content.js

```diff
@@ -21,5 +21,6 @@ export async function getEditedPostContent() {
 		throw new Error( 'Unexpected zero-width space character in editor content.' );
 	}
 
-	return content;
+	// Encode non breaking spaces so they are visible in snapshots.
+	return content.replace( /\u00a0/g, '&nbsp;' );
```

Comment:
Kinda seems like we should throw if there's one present, shouldn't we? Like in the lines previous for zero-width space characters.

Reply:
Hm, I think it's better to just make them visible. Ideally we'd have some tests with the keyboard inserting non breaking spaces, but haven't figured out how yet.

Reply:
At the end of the day, non breaking spaces are allowed, just shouldn't be inserted unexpectedly.

Reply:
Yeah, my thinking is that the NBSP are much less expected, just like with the ZWSP, but can easily slip in (case in point: the originating issue).